mlir/test/Dialect/Vector/vector-rewrite-narrow-types.mlir

```diff
@@ -300,16 +298,32 @@ func.func @aligned_trunci_2d(%a: vector<8x32xi32>) -> vector<8x32xi4> {
 // CHECK-NOT:       vector.shli
 // CHECK-NOT:       vector.ori
 // CHECK:           arith.trunci
```

Comment:
Shouldn't this `arith.trunci` be gone? Isn't that the point of `RewriteAlignedSubByteIntTrunc`?

Reply:
This is i32 to i4. So it goes trunci i32 to i8 then i8 to i4 (via deinterleave).

Reply:
OK, thanks! Looks like this this is missing types. You can also add `CHECK-NOT: trunci` after the first `CHECK: trunci`

Reply:
New CHECK-NOT clause added!